packages/monitors/README.md

```diff
@@ -0,0 +1,31 @@
+# @uma/monitors
+
+This package contains four UMA monitor bot implementations: `BalanceMonitor`, `ContractMonitor`, `CRMonitor`, and `SyntheticPegMonitor`. The monitor bots are used to monitor the UMA ecosystem for key events. They have a number of monitor modules built into them that enable real time reporting on key events for a given EMP contract.
+
+For more information about running a monitor bot, see the [docs](https://docs.umaproject.org/developers/bots).
+
+## Installing the package
+
+```bash
+yarn add @uma/monitors
+```
+
+## Running the monitors
+
+The config below will start up a monitor bot that will: (1) send messages when new liquidations, alerts, or disputes occur and (2) fire if there is large volatility in the synthetic or price of the underlying. It won't report on any wallet or CR monitoring as no params have been defined.
```

Comment:
It would be great to add a little documentation around how you can add a slack webhook or pagerduty info to get the monitor to alert you outside of the console. We don't have to go into great detail, but would be nice to mention the env variables (in a separate section).